packages/zk-connect-react/README.md

```diff
@@ -3,26 +3,72 @@
   <img src="https://static.sismo.io/readme/top-main.png" alt="Logo" width="150" height="150" style="borderRadius: 20px">
 
   <h3 align="center">
-    zkConnect React
+    sismoConnect React
   </h3>
 
   <p align="center">
     Made by <a href="https://docs.sismo.io/" target="_blank">Sismo</a>
   </p>
   
   <p align="center">
+    <a href="https://twitter.com/sismo_eth" target="_blank">
+        <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white"/>
+    </a>
     <a href="https://discord.gg/sismo" target="_blank">
         <img src="https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white"/>
     </a>
-    <a href="https://twitter.com/sismo_eth" target="_blank">
-        <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white"/>
+    <a href="https://builders.sismo.io" target="_blank">
+        <img src="https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white"/>
     </a>
   </p>
 </div>
 
-zkConnect is a privacy-preserving single sign-on method for applications. Once integrated, applications can request private, granular data from users, while users can authenticate and selectively reveal their data thanks to zero-knowledge proofs (ZKPs). 
+sismoConnect is a privacy-preserving single sign-on method for applications. Once integrated, applications can request private, granular data from users, while users can authenticate and selectively reveal their data thanks to zero-knowledge proofs (ZKPs). 
+
+The sismoConnect React package is a wrapper of the sismoConnect client package which is a package build on top of the Sismo Data Vault app (the prover) to easily request proofs from your users.
+
+You can see the documentation of the sismoConnect React package [here](https://docs.sismo.io/sismo-docs/technical-documentation/zkconnect/zkconnect-react-request).
+
+## Installation
```

Comment:
Can you put this in the gitbook? By doing so, we will only have one source of truth to update not severals